arbitrum-docs/how-arbitrum-works/02-transaction-lifecycle.mdx

```diff
@@ -1,4 +1,26 @@
-# Overview: The Lifecycle of an Arbitrum Transaction
+---
+title: Transaction Lifecycle
+---
+
+## Sequencing, Followed by Deterministic Execution
```

Comment:
This could be the section of the page, to align with the 4 big ideas mentioned in the Introduction.
This first part serves as a "In a nutshell" (a big nutshell though) section to the rest of the page, since it's summarizing the lifecycle of a transaction.

The second part of the page goes into much more detail than this, so as a reader, I'd like to know that this bit is going to introduce me to what's going to be explained more in detail afterwards.

Reply:
I'll rename the page to "Sequencing..." 

As to separate the first and second halves of the doc, currently the second section is "Overview: The lifecycle..." since this is a deeper technical breakdown, I'm going to rename this to: "Deep dive: The lifecycle" to delineate the "summary/top" from the "technical/bottom"